OpenEdXMobile/src/main/java/org/edx/mobile/services/CourseManager.java

```diff
@@ -134,4 +139,26 @@ public boolean apply(CourseComponent courseComponent) {
             }
         });
     }
+
+    /**
+     * Obtain the enrolled courses data from app level cache and then find the specified course.
```

Comment:
Function doc is not properly indented.
Pro Tip: Use "Option + Command + L" short cut key.
Also content of the comment needs to be improved as well